controller/lib/core/blower_fsm.cpp

```diff
@@ -15,8 +15,40 @@ limitations under the License.
 
 #include "blower_fsm.h"
 #include "controller.h"
+#include "vars.h"
 #include <algorithm>
 
+// dbg_pa_* are pressure assist configuration vars.
+//
+// These are read but never modified here.
+
+// TODO: This should be configurable from the GUI.
+static DebugFloat dbg_pa_flow_trigger("pa_flow_trigger",
+                                      "pressure assist flow trigger (ml/s)",
+                                      200);
+
+// TODO: Is 250ms right?  Or can it be a fixed value at all; should it depend
+// on the RR or something?
+static DebugFloat
+    dbg_pa_min_expire_ms("pa_min_dwell_ms",
+                         "minimum amount of time after ventilator exits PIP "
+                         "before we're eligible to trigger a breath",
+                         250);
+
+// pa_fast_avg_alpha and pa_slow_avg_alpha were tuned for a control loop that
+// runs at a particular frequency.  If the control loop gets slower, the alpha
```

Comment:
Code below seems to suggest that these variables actually will not need tuning if the loop period changes